automation/rpm.sh

```diff
@@ -20,7 +20,7 @@ mkdir -p ${EXPORT_DIR}
 
 cp $PWD/lib/vdsm/api/vdsm-api.html "${EXPORT_DIR}"
 
-find ~/rpmbuild \
+find $PWD/build \
```

Comment:
$(RPM_TOPDIR)

Reply:
It shouldn't be available from within this script?

Reply:
Ah, right, sorry for noise.